.gitignore

```diff
@@ -5,6 +5,9 @@
 gae_runtime/*
 third_party/*
 core/templates/prod/*
+static/fonts/*
```

Comment:
The third_party library does get uploaded during deployment. (In general, anything in the oppia/ folder gets uploaded.)

Putting everything that's generated in third_party/generated sounds good to me. I would lean towards doing the same for fonts (and not creating static/fonts).